client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -14,18 +13,13 @@ const AuthFooter = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <Box
-      p={1}
-      display="flex"
-      justifyContent="flex-end"
-      alignSelf="flex-end"
-      marginRight={5}
-      className={classes.authHeader}
-    >
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Button component={Link} to={linkTo} color="inherit" className={classes.accBtn} variant="contained">
-        {btnText}
-      </Button>
+    <Box p={1} display="flex" justifyContent="center" alignSelf="center" className={classes.authHeader}>
+      <Typography className={classes.accAside}>
+        {asideText + '\xa0'}
+        <Link to={linkTo} color="inherit" className={classes.accBtn}>
```

Comment:
```suggestion
        <Link to={linkTo} color="inherit" className={classes.authLink}>
```
Nit: Why don't we change the name of the CSS class since this is no longer a button